src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.split(';')
+    .map(styleLine => styleLine.split(':'))
+    .filter(styleLine => styleLine.length === 2)
+    .map(styleLine => styleLine
+      .map(styleProperty => styleProperty.trim())
+    );
```

Comment:
If you chain array methods they all should start on new line with correct indentation.
```suggestion
  const styleLines = sourceString
    .split(';')
    .map(styleLine => styleLine.split(':'))
    .filter(styleLine => styleLine.length === 2)
    .map(styleLine => styleLine
      .map(styleProperty => styleProperty.trim())
    );
```